### ISSUE TITLE
Supporting autoform 5.0

### ISSUE THREAD

**Issue by aldeed** [yogiben/meteor-admin#159]
Hi @mpowaga @yogiben, I was going to do a PR to update to support autoform 5.0, just released, but the code pushed to github seems to be a bit disorganized right now. The master branch doesn't seem to represent what is actually published since it has tabular pkg code which I don't think the latest published release, 1.1.0, has.

If you can update master or some branch to match the published 1.1.0 release, I can do the PR. I may be able to help finish the switchover to tabular soon, too, if you're still planning to release that.

**Comment by aldeed** [comment]
Got master running and did a PR against that. See #161

**Comment by hiphapis** [comment]
Hi @aldeed long time no see ;)
anyway. this project is not running anymore. just my guess.
i want to wake up too

**Comment by obiwarn** [comment]
Yes, it is very quiet here.

Maybe it is time for you @aldeed to take over? There is so much technology from you in here anyways.

What about an  aldeed:admin? I would love to see that :-)

**Comment by aldeed** [comment]
afaik, they are still maintaining. Just a bit behind. There is also now competition from [kaoskeya:admin](https://atmospherejs.com/kaoskeya/admin). I already have too many packages to properly maintain them, so I'd prefer to help out these admin packages rather than write my own.

**Comment by obiwarn** [comment]
Oh, thank you for the hint to kaoskeya. Looks promising.

**Comment by rustanacexd** [comment]
yeah this meteor-admin seems to be very buggy at my end too. Thanks for the hint to kaoskeya. Looks very cool

**Comment by theroadrunnershow** [comment]
Hi, I am looking for a fix for this issue:
https://github.com/aldeed/meteor-autoform/issues/785

But looks like there is no way without updateing to v5.1.0

So, any other workaround?

**Comment by brylie** [comment]
@yogiben any status update? I get the following error when installing this package:

```
Potentially incompatible change required to top-level dependency:
aldeed:autoform 4.2.2, was 5.1.2.
Constraints on package "aldeed:autoform":
* aldeed:autoform@4.2.2 <- yogiben:admin 1.0.9
```

**Comment by aldeed** [comment]
@brylie, you may just need to do `meteor update --allow-incompatible-update`

**Comment by theroadrunnershow** [comment]
@Eric, but there is a chance something in yogiben's package might break?

**Comment by brylie** [comment]
@aldeed would this roll autoform version back to 4.x? I am happy to stay with 5.x :-)

**Comment by aldeed** [comment]
No, I think the latest Meteor release will sometimes not update packages to a new major release unless you add that `--allow-incompatible-update` option.

**Comment by brylie** [comment]
OK, I was hesitant because of the following line:

```
aldeed:autoform 4.2.2, was 5.1.2.
```

**Comment by aldeed** [comment]
You can always change to the versions you want in the versions file and see if it will run. The new `--allow-incompatible-update` isn't really documented and the error messages don't make sense to me. I always feel like they say the opposite of what they mean.

I think there is some issue with the current yogiben:admin release, though. I'm currently using my fork of it until that's resolved.

**Comment by yogiben** [comment]
Hey guys,

I'm sorry for the slowness in maintenance. I'm teaching Meteor [full time in Ghana](https://github.com/yogiben/meteor-admin) and this is taking up too much time.

If anyone wants to get involved, please let me know.

Updating to the latest release allows `'aldeed:autoform@4.2.2 || 5.1.2'`
